src/main/resources/templates/nonMedicalUser.html

```diff
@@ -3,16 +3,18 @@
 
 <head>
   <title>Peace Of Mind</title>
+  <link href="/css/styles.css" rel="stylesheet">
 </head>
 
 <body class="container">
   <div>
     <header>
       <h1>Peace of Mind</h1>
       <nav>
-        <h2>
           <a th:href="@{/non-medical-user-home(id=${nonMedicalUser.id})}">Home</a>
-          <h2>
+          <a th:href="@{/non-medical-user-home(id=${nonMedicalUser.id})}">Account</a>
```

Comment:
Hey Simon, 
Let's leave these as is for now, these links you've added all point to the home page and I don't want us risking that they'll get overlooked as we progress

Reply:
if you want to add a placeholder link that simply says something like "Placeholder - Medications" that's cool too, that'll give us a good reminder to add it in when we have everything wired up :)